lib/audit/catalog_to_moab.rb

```diff
@@ -66,32 +66,33 @@ def check_catalog_version
     # TODO: anything special if preserved_copy.status is not OK_STATUS? - see #480
 
     unless online_moab_found?(druid, storage_dir)
+      update_status(PreservedCopy::ONLINE_MOAB_NOT_FOUND_STATUS)
       results.add_result(PreservedObjectHandlerResults::ONLINE_MOAB_DOES_NOT_EXIST)
       results.report_results
+      preserved_copy.save!
       return
     end
 
     moab_version = moab.current_version_id
     catalog_version = preserved_copy.version
     if catalog_version == moab_version
+      set_status_as_seen_on_disk(true) unless preserved_copy.status == PreservedCopy::OK_STATUS
       results.add_result(PreservedObjectHandlerResults::VERSION_MATCHES, preserved_copy.class.name)
       results.report_results
     elsif catalog_version < moab_version
+      set_status_as_seen_on_disk(true) unless preserved_copy.status == PreservedCopy::OK_STATUS
```

Comment:
should we always just set status in this case, regardless of current catalog status?  see skipped test...

Reply:
See below - basically:  I'm ok with whatever;  it's kinda up to @julianmorley, no?

Reply:
discussed in story time, @julianmorley says fine to just set the status regardless when C2M finds that the version on disk has increased compared to the catalog.